package.json

```diff
@@ -59,6 +63,7 @@
     "@spruceid/siwe-parser": "^2.0.0",
     "@stitches/react": "^1.2.8",
     "@storybook/react": "^7.6.0",
+    "@tanstack/react-query": "^5.18.0",
```

Comment:
is this required?

Reply:
I don't think it's needed anymore it was used for when I was using wagmi to connect.
I am going to remove it